api-authorizations/pom-template.xml

```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>com.normation.plugins</groupId>
     <artifactId>plugins-parent</artifactId>
-    <version>${rudder-branch}-${parent-version}</version>
+    <version>${parent-version}</version>
```

Comment:
I don't understand that. Plugin parent must be by branch, since it holds all dependencies. Do you want to change the way it is versionned ?

Reply:
In fact, I don't understand at all. The sed that used to replace it by the plugin-parent version is deleted, and I don't understand where it comes from now.

Reply:
it's still by branch, it just that i refactored to have less variables and they are now treated by the makefile depending on the target called

Reply:
see here

Reply:
See here: https://github.com/Normation/rudder-plugins/pull/403/files#diff-a288abe3cd89222034a3c163912d7d6769a3bddd75d2223b51ca02d92386cceeR28

Reply:
I just removed useless variables (like nightly variables) and now the nightly and snapshot are put on the right place whether you call the correct make target ( all / nightly or next) the makefille will now create a put the right value at the right place instead of depending on a value from a conf file that was either wrong or needed to be manipulated

Reply:
ok, the definition part was missing in the revesion I reviewed, now I understand, thanks.